[2022-06-03 02:07:50,444] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T02:07:41.671826+00:00 [queued]>
[2022-06-03 02:07:50,465] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T02:07:41.671826+00:00 [queued]>
[2022-06-03 02:07:50,466] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 02:07:50,468] {taskinstance.py:1358} INFO - Starting attempt 1 of 2
[2022-06-03 02:07:50,468] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 02:07:50,518] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): processing_data> on 2022-06-03 02:07:41.671826+00:00
[2022-06-03 02:07:50,533] {standard_task_runner.py:52} INFO - Started process 858 to run task
[2022-06-03 02:07:50,545] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'nyc_bus_breakdown_pipeline', 'processing_data', 'manual__2022-06-03T02:07:41.671826+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/nycbus_etl.py', '--cfg-path', '/tmp/tmpr453_zqf', '--error-file', '/tmp/tmp7_zaheq8']
[2022-06-03 02:07:50,548] {standard_task_runner.py:80} INFO - Job 126: Subtask processing_data
[2022-06-03 02:07:50,649] {task_command.py:370} INFO - Running <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T02:07:41.671826+00:00 [running]> on host b7655f13eea6
[2022-06-03 02:07:50,782] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=nyc_bus_breakdown_pipeline
AIRFLOW_CTX_TASK_ID=processing_data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T02:07:41.671826+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-03T02:07:41.671826+00:00
[2022-06-03 02:07:52,321] {base.py:68} INFO - Using connection ID 's3_conn' for task execution.
[2022-06-03 02:07:52,323] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-06-03 02:07:54,220] {python.py:173} INFO - Done. Returned value was: None
[2022-06-03 02:07:54,279] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=nyc_bus_breakdown_pipeline, task_id=processing_data, execution_date=20220603T020741, start_date=20220603T020750, end_date=20220603T020754
[2022-06-03 02:07:54,456] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-03 02:07:54,599] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
